inference-engine/tests/functional/plugin/cpu/shared_tests_instances/behavior/plugin/core_integration.cpp

```diff
@@ -99,4 +99,8 @@ INSTANTIATE_TEST_SUITE_P(
 INSTANTIATE_TEST_SUITE_P(
         smoke_IEClassLoadNetworkTest, IEClassLoadNetworkTest,
         ::testing::Values("CPU"));
-} // namespace
\ No newline at end of file
+
+// IE Class Load network to default device
+INSTANTIATE_TEST_SUITE_P(smoke_Auto_DefaultLoadTests, IEClassLoadNetworkToDefaultPluginTest,
+            ::testing::Values(""));
```

Comment:
rather than pass "" as the device name... shall the **original declaration of the LoadNetwork** be changed so its last parameter (device name) has the default value of ""?

Reply:
yeah, better to, shall I change it in this PR or do I need a new jira for this change?

Reply:
this PR is ok, adding @ilya-lavrenov as this is general API change (and e.g. we shall change only the new, OV 2.0 APIs) with having a default device name

Reply:
@myshevts @ilya-lavrenov I would like to clarify the cases of device name in AUTO and related PRs.
1. "AUTO" name as other HW plugin. For example, LoadNetwork(network, "AUTO"). This case is enabled in master brach.
2. Empty name "". For example, LoadNetwork(network, ""). This PR will validate case 2.
3. Default name. For example, LoadNetwork(network). PR 8494 (https://github.com/openvinotoolkit/openvino/pull/8494) will enable case 3.

Reply:
I believe that the 2 is redundant.
on the 3 the only question is whether we want to equip the new (OV 2.0) API only, or togther with old API.
notice that the  PR 8494 (#8494)  changes only old API

Reply:
Yes. After 3 is enabled, 2 can be removed.